cypress/support/server.ts

```diff
@@ -1940,3 +1942,59 @@ export const mockUpdatePassword = (
     },
   ).as('updatePassword');
 };
+
+export const mockGetItemFavorites = (
+  itemFavorites: ItemFavorite[],
+  shouldThrowError: boolean
+): void => {
+  cy.intercept(
+    {
+      method: HttpMethod.GET,
+      url: `${API_HOST}/${GET_FAVORITE_ITEMS_ROUTE}`,
+    },
+    ({ reply }) => {
+      if (shouldThrowError) {
+        return reply({ statusCode: StatusCodes.BAD_REQUEST });
+      }
+
+      return reply(itemFavorites);
+    },
+  ).as('getFavoriteItems');
+};
+
+export const mockAddFavorite = (
+  _items: ItemForTest[],
+  shouldThrowError: boolean,
+): void => {
+  cy.intercept(
+    {
+      method: HttpMethod.POST,
+      url: new RegExp(`${API_HOST}/${GET_FAVORITE_ITEMS_ROUTE}/${ID_FORMAT}`),
```

Comment:
It doesn't have to be changes, but if possible you can use the API endpoints from query-client, so it doesn't break if we change the endpoint route.

Reply:
The problem is that the endpoint builder takes a UUID to build the url, and the ID_FORMAT is a string